src/gui/topbar_subwindows/gui_trade_window.hpp

```diff
@@ -492,7 +459,7 @@ class trade_commodity_entry : public window_element_base {
 		} else if(name == "selling_indicator") {
 			return make_element_by_type<commodity_stockpile_indicator>(state, id);
 		} else if(name == "automation_indicator") {
-			return make_element_by_type<commodity_ai_indicator>(state, id);
```

Comment:
Wasn't this used to indicate whether the stockpile was filling or not? Or something like that?

Reply:
> Wasn't this used to indicate whether the stockpile was filling or not? Or something like that?

nope